### PR TITLE
Create Mobile Portfolio

### PR DESCRIPTION
**Pull request by Wolfmikerson**
This pull request includes:

1- Configure Linters
2- Add README.md
3- Create my professional portfolio with HTML & CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio website</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <!-- Header Section-->
+  <header>
+    <nav class="navbar-top-contener"></nav>
```
Comment (AshakaE) [inline index.html]: - [ ] Please remove this content, it is not needed 🙏

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <nav class="navbar-contener default-display-flex">
+      <div class="logo"><a class="logo-font default-custom-fonts" href="#">My Logo</a></div>
+      <div class="menu"><img src="images/menu_icon.png" alt=""></div>
```
Comment (AshakaE) [inline index.html]: - [ ] Please replace the `div` tag with a `nav` tag 🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <meta charset="UTF-8">
+    <title>My Portfolio website</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <!-- Header Section-->
+  <header>
+    <nav class="navbar-top-contener"></nav>
+    <nav class="navbar-contener default-display-flex">
```
Comment (AshakaE) [inline index.html]: [OPTIONAL]
- [ ] I suggest removing this `nav` element and applying the styles here directly to the `header` element 🤷‍♂️

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+          <h2 class="support-text-2 default-custom-fonts lets-text-font">LETS CONNECT</h2>
+          <ul class="default-display-flex default-list-style">
+            <li class="display-list-item"><a href="https://twitter.com/MikersonWolf" title="Visit my twitter" target="_blank" rel="noopener noreferrer"><img src="images/Twitter.jpeg" alt=""></a></li>
+            <li class="display-list-item"><a href="https://www.linkedin.com/in/wolf-m-jacques-27a38612a/" title="Visit my linkedin" target="_blank" rel="noopener noreferrer"><img src="images/linkedn.jpeg" alt=""></a></li>
+            <li class="display-list-item"><a href="#" title="Visit my Medium"><img src="images/Median.jpeg" alt=""></a></li>
+            <li class="display-list-item"><a href="https://github.com/Wolfmikerson" title="Visit my Github" target="_blank" rel="noopener noreferrer"><img src="images/Github.jpeg" alt=""></a></li>
+            <li class="display-list-item"><a href="#" title="Visit my Angellist"><img src="images/Angellist.jpeg" alt=""></a></li>
+          </ul>
+        </div>
+    </section>
```
Comment (AshakaE) [inline index.html:29]: - [ ] Please add some left and right padding to this section

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Wolfmikerson  🙋‍♀️, 

Good job so far! 👌 🚀 
There are some issues that you still need to work on to go to the next project but you are almost there!

#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  margin-bottom: 136px;
+  padding: 0;
+}
+
+/* nav bar styling */
+.navbar-contener {
+  width: 100%;
+  padding: 10px 0;
+
+  /* justify-content: space-between; */
```
Comment (SajjadAhmad14) [inline css/style.css]: [OPTIONAL] Please remove commented code. It makes the whole code base unclean.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Wolf Jacques, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**